java/tests/com/google/privacy/differentialprivacy/BoundedMeanTest.java

```diff
@@ -70,6 +71,20 @@ private static void mockLongNoise(Noise noise, long value) {
     when(noise.getMechanismType()).thenReturn(MechanismType.GAUSSIAN);
   }
 
+  private static void mockDoubleConfInt(Noise noise, ConfidenceInterval confInt) {
```

Comment:
I am not sure we need to pass noise. Someone can pass actual noise implementation and it will break everything.

Also why we set LAPLACE here? when in setup we use gauissian?

Reply:
I have set it to GAUSSIAN instead.
To remove the noise parameter for that reason, maybe edit this as well ?
https://github.com/google/differential-privacy/pull/54/files#diff-4b1a707aed1d5c6296064d1daa2ee6beR62-R71

Reply:
do we need to set it to GAUSSIAN here if it was set in setUp method?
I would remove it , as well as the noise parameter (in both mock**Noise and mock**ConfInt)

Reply:
We only need to set it up once in the setUp method since it is required in checks
https://github.com/google/differential-privacy/blob/2ccce87b6f4508efd991732f7227dcda3cd39fb2/java/main/com/google/privacy/differentialprivacy/BoundedMean.java#L246
and implicitly in BoundedSum:
https://github.com/google/differential-privacy/blob/2ccce87b6f4508efd991732f7227dcda3cd39fb2/java/main/com/google/privacy/differentialprivacy/BoundedSum.java#L384

I removed the noise parameter, made the mock object static and just set it once in setUp method.